src/azure-cli/azure/cli/command_modules/vm/_help.py

```diff
@@ -531,6 +531,9 @@
         --target-regions westus=2=standard_lrs eastus=3=standard_zrs \\
         --gallery-name MyGallery --gallery-image-definition MyImage \\
         --managed-image imageInTheSameResourceGroup
+  - name: Add a new image version with target regions and customer managed keys for encryption.
+    text: >
+        az sig image-version create -g MyResourceGroup --gallery-image-version 1.0.0 --target-regions westus=2=standard_--target-region-encryption DiskEncryptionSet1,0,DiskEncryptionSet2 --gallery-name MyGallery --gallery-image-definition MyImage --managed-image imageInTheSameResourceGroup
```

Comment:
for `target-region-encryption`, how about split it into `-osdisk-target-region-encrytion` and `datadisk-target-region-encryption`, your description is very clear, just think from user perspective how to avoid error

Reply:
This parameter receives encryption settings for multiple regions. Each encryption setting contains disk encryption set for OS disk and data disks.
I use one region in this example because service is not ready for setting encryption for multiple regions. But it should has the ability according to swagger spec and example.